src/main/java/net/glowstone/net/handler/play/inv/NameItemHandler.java

```diff
@@ -0,0 +1,12 @@
+package net.glowstone.net.handler.play.inv;
+
+import com.flowpowered.network.MessageHandler;
+import net.glowstone.net.GlowSession;
+import net.glowstone.net.message.play.inv.NameItemMessage;
+
+public final class NameItemHandler implements MessageHandler<GlowSession, NameItemMessage> {
+    @Override
+    public void handle(GlowSession session, NameItemMessage message) {
+        //TODO: handle packet
+    }
+}
```

Comment:
In `PluginMessageHandler`, you will find the `MC|ItemName` handler that needs to be moved here (make sure you update your branch first).